pkg/operator/status/status_controller.go

```diff
@@ -113,24 +114,6 @@ func (c StatusSyncer) sync() error {
 	}
 	clusterOperatorObj := originalClusterOperatorObj.DeepCopy()
 
-	// if we are unmanaged, force everything to Unknown.
-	if detailedSpec.ManagementState == operatorv1.Unmanaged {
```

Comment:
skip this block if the operator is "AlwaysOn"

Reply:
why? lets set this in management state controller?

Reply:
nvmd.